src/ast/mod.rs

```diff
@@ -808,6 +808,21 @@ pub enum Expr {
     },
     /// Scalar function call e.g. `LEFT(foo, 5)`
     Function(Function),
+    /// Arbitrary expr method call
+    ///
+    /// Syntax:
+    ///
+    /// `<arbitrary-expr>.<arbitrary-expr>.<arbitrary-expr>...`
+    ///
+    /// > `arbitrary-expr` can be any expression including a function call.
```

Comment:
Ah if i understand correctly I think this comment might not be accurate. My understanding is that first expression is indeed an arbitrary expression but subsequent expressions must be function calls?